notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -42,6 +42,46 @@ define([
         this.edit_shortcuts = new keyboard.ShortcutManager(undefined, options.events, this.actions, this.env);
         this.edit_shortcuts.add_shortcuts(this.get_default_common_shortcuts());
         this.edit_shortcuts.add_shortcuts(this.get_default_edit_shortcuts());
+
+
+        this.config = options.config;
+        var that = this;
+
+        this.config.loaded.then(function(){ 
+            var edit_unbind;
+
+            try {
+                edit_unbind = that.config.data.keys.edit.unbind;
+            } catch (e) {
+                if (e instanceof TypeError) {
+                    edit_unbind = [];
+                } else {
+                    throw e;
+                }
+            }
+
+            edit_unbind.forEach(function(u){that.edit_shortcuts.remove_shortcut(u);});
```

Comment:
No, I actually tried, and it was calling the prototype in the wrong context, telling me that `undefined has no attribute i_dont-remember_what` my guess is that forEach will actually `.bind(..)` the callback to something else.